### DISCUSSION TITLE
How to unmount table expandable component when expand button is clicked?

### DISCUSSION THREAD

**Discussion by dipbazz** [ant-design/ant-design#40323] [Q&A] [answered]
I want to completely unmount the expandable component once it is opened and then closed. For now when I click the expand button it mounts and display the component but when I click again to close the expanded component, It only disappear  from the view but it is not unmounted. Because of which if I have any code that's running on that component it will execute while the component is not on the view.

**Comment by zombieJ** [comment, marked as answer]
Seems HOC expand content that when item is not in expand key render null to handle this.
